static/src/stylesheets/pasteup/publish.js

```diff
@@ -0,0 +1,98 @@
+// Don't run this file manually – to publish a new release run `make pasteup` from `frontend/`
+
+var Promise = require("bluebird");
+var writeFile = Promise.promisify(require("fs").writeFile);
+var exec = require('child_process').exec;
+var del = require('del');
+var megalog = require('megalog');
+
+var authTokenKey = 'npm.guardian.developers.authToken';
+
+var npmCredentials = (function () {
+    var propertiesFile = require('path').join(require('home-dir')(), '.gu', 'frontend.properties');
+    var npmAuthToken = require('java-properties').of(propertiesFile).get(authTokenKey);
+    return npmAuthToken;
+})();
+
+function setCredentials(authToken) {
+    return writeFile('.npmrc', [
+        'init.author.name=Guardian developers',
+        'init.author.email=dotcom.engineers@theguardian.com',
+        '//registry.npmjs.org/:_authToken=' + authToken,
+        'registry=https://registry.npmjs.org/'
+    ].join('\n'));
+}
+
+function getReleaseType() {
+    return new Promise(function(resolve, reject) {
+        require("inquirer").prompt([{
```

Comment:
Slight style suggestion, I'd usually put all requires / imports at the top of the file. Makes it easier to see what dependencies are used per module.